### ISSUE TITLE
Jetpack Comments: Allow local translations

### ISSUE THREAD

**Issue by alexjgustafson** [Automattic/jetpack#5197]
Currently Jetpack Comments receives all its translations through GlotPress before serving up the comment box in an iframe. If a site owner feels a translation is incorrect, they have no way of adjusting this but to wait on/offer a translation to GlotPress. 

But what if GlotPress' translation is not what the owner wants? What if they do offer a translation and it is rejected by a GlotPress editor? Then they have no way of customizing these phrases on their site.

Could we add a hook that allows a site owner to locally customize the translations/phrases within the Jetpack comment box?

**Comment by alexjgustafson** [comment]
#2845998-t

**Comment by jeherve** [comment]
I think this will be a `wontfix` for now. This doesn't only affect Comments; it's also true for Likes, Subscriptions, Stats, and all other modules and options available on WordPress.com, or in an iFrame served from WordPress.com. WordPress doesn't currently have a system allowing one to serve custom translations for only one site on a network of sites, and to serve those translations from a custom location outside of the network.

> What if they do offer a translation and it is rejected by a GlotPress editor?

I feel this is what needs to be addressed here. Why would the translation be rejected, if it's a better / more accurate one? Is there something we could do to surface validators more, like on WordPress.org, to allow translators to get in touch with validators to discuss such issues?
